test/protocol/ProtocolInitializationHandlerTest.js

```diff
@@ -458,20 +467,6 @@ describe("ProtocolInitializationHandler", async function () {
       );
     });
 
-    it("Should emit MaxPremintedVouchersChanged event", async function () {
```

Comment:
Just related to my other comment (in contracts/protocol/facets/ProtocolInitializationHandlerFacet.sol).
If we bring `MaxPremintedVouchersChanged` back, this test should also be put back.